frontend/composables/use-validators.ts

```diff
@@ -3,11 +3,18 @@ const EMAIL_REGEX =
 
 const URL_REGEX = /[-a-zA-Z0-9@:%._+~#=]{1,256}.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/;
 
+validate(what: string, value: string): boolean {
+	const response =  fetch(`/api/validators/${what}/${value}`);
+  const data = response.json();
+	return ! data.error;
+}
+
```

Comment:
need to use the function keyword 

```ts
function validate(what: string, value: string): boolean {
	const response =  fetch(`/api/validators/${what}/${value}`);
  const data = response.json();
	return ! data.error;
}
```

Reply:
Well, won't work anyway. sorry I accidentally added the fronting files.